deploy/helm/magda-core/charts/authorization-db/templates/migration.yaml

```diff
@@ -20,4 +20,4 @@ spec:
         env:
         - name: DB_HOST
           value: authorization-db
-        {{- template "magda.postgres-migrator-env" . }}
\ No newline at end of file
+        {{- template "magda.postgres-migrator-env" . }}
```

Comment:
Seems no changes to this line?

Reply:
My VSCode YAML linter added a newline at the end of this :smile: